scripts/lint.sh

```diff
@@ -94,4 +94,16 @@ if [ ! -z "$output" ]; then
 	exitcode=1
 fi
 
+# ShellCheck linting
+for file in `find . -name "*.sh"`
+do
+	if ! [[ $file == ./third_party/* ]] && ! [[ $file == ./cmake-* ]] && ! [[ $file == ./clang-* ]];
```

Comment:
Is `! [[ $file == ./clang-* ]]` used to exclude clang build directories? Build directories can be named arbitrarily so they wouldn't be generally excluded in any cases.

How about including certain directories instead? For Hyrise, scripts are expected to be located in the scripts directory. In addition, `install_dependencies.sh` is located in the root directory.

How about only considering sh-files located in the project's root or `scripts` directory?

Reply:
This is the better way, done!

Reply:
When you're only accessing sh-files in the scripts directory, you can remove all the checks as Marcel hinted.

Reply:
In addition, I would include `install_dependencies.sh` (or .sh files located in the root directory in general).